gubled/config/config.go

```diff
@@ -73,6 +74,24 @@ var (
 			Envar("GUBLE_GCM_WORKERS").
 			Int(),
 	}
+	// Cluster settings related to activating the cluster module
+	Cluster = struct {
+		NodeID   *int
+		NodePort *int
+		Remotes  *[]*url.URL
+	}{
+		NodeID: kingpin.Flag("node-id", "This guble node's own ID (used in cluster mode): a strictly positive integer number which must be unique in cluster").
```

Comment:
however, to eliminate any misunderstanding, I will validate it right from the start